### PR TITLE
Fix/bryan/correct deps and travis

### PR DESCRIPTION
**Pull request by ghost**
I didn't see - 

https://github.com/esl/exometer_core/blob/00407ddca96a3b6413a49fe86a68bd62fd6c2e5e/src/exometer_info.erl#L53-L55

https://github.com/esl/exometer_core/blob/00407ddca96a3b6413a49fe86a68bd62fd6c2e5e/src/exometer_info.erl#L29

Because I removed the deps - but didn't perform a 'rebar3 unlock' to update the lock file. 

So.. the parse_trans and bear deps need to go back in for now, otherwise the common test will fail.

The parse trans stuff is minimal, as far as I can see, it's just compensating for the fact that older erlang doesn't have a map datatype. Perhaps we can switch from record to map later and fix this. 

Bear seems to be used to verify various statistics, it's very old school, hasn't been updated in 7 years and doesn't provide much in the way of documentation. 

Properly getting rid of parse_trans would allow this library to be used in current and future Elixir projects - but that's a task for another day.

### CODE REVIEW

**Comment by ghost** [comment]
@kzemek - my previous p/r was incorrect - this fixes it

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 sudo: false
 language: erlang
-before_script:
-    - wget https://github.com/erlang/rebar3/releases/download/3.6.1/rebar3
-    - chmod +x rebar3
 script:
-    - ./rebar3 eunit
-    - ./rebar3 ct
```
Comment (michalwski) [inline .travis.yml:8]: Travis CI provides too old rebar3 version. That's why we need to manually download 3.6.1 and use it us before. The old version doesn't work with Erlang/OTP 21 correctly. @bryanhuntesl  please revert this and above changes.

Comment (ghost) [reply .travis.yml:8]: @michalwski - 

fixed by  - https://github.com/esl/exometer_core/pull/6/commits/111b828bb6c79debf58756e992de1775ed7f8505

pinged travis existing request to upgrade rebar3 - https://github.com/travis-ci/travis-ci/issues/9830#issuecomment-418301427

Comment (ghost) [reply .travis.yml:8]: Good to go ?

Comment (michalwski) [reply .travis.yml:8]: Yes, I'm merging it.

Comment (ghost) [reply .travis.yml:8]: Thanks @michalwski

**Pull request merged**